tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -137,7 +137,7 @@ private void serveWaitingPage(HttpServletRequest request, PrintWriter out, Surve
          * the front end could be in charge of the JavaScript.
          */
         String url = request.getContextPath() + request.getServletPath();
-        out.write("<html lang='" + SurveyMain.TRANS_HINT_LOCALE.toLanguageTag() + " class='claro'>\n");
+        out.write("<html lang='" + SurveyMain.TRANS_HINT_LOCALE.toLanguageTag() + "' class='claro'>\n");
```

Comment:
@btangmu this was an old bug, causes an invalid HTML page during loading

Reply:
good catch!